.storybook/preview.tsx

```diff
@@ -47,12 +48,16 @@ const withThemeDocs: DecoratorFn = ({children, context}) => {
   }
 
   return (
-    <ThemeProvider theme={currentTheme}>
-      <GlobalStyles isDark={isDark} theme={currentTheme} />
-      <PreviewGlobalStyles theme={currentTheme} />
-      <DocsContainer context={context}>{children}</DocsContainer>
-      <TableOfContents />
-    </ThemeProvider>
+    <Fragment>
+      <DocsContainer context={context}>
+        <GlobalStyles isDark={isDark} theme={currentTheme} />
+        <PreviewGlobalStyles theme={currentTheme} />
+        <ThemeProvider theme={currentTheme}>{children}</ThemeProvider>
+      </DocsContainer>
+      <ThemeProvider theme={currentTheme}>
+        <TableOfContents />
```

Comment:
Is it ok that this is not part of DocsContainer? This is a bit awk, but not a big deal

Reply:
Should be okay. `DocsContainer` is just a wrapper around the main content. There's another wrapper (`#docs-root`) surrounding both it and `<TableOfContents />`.